Resources/Prototypes/Entities/Objects/Devices/pinpointer.yml

```diff
@@ -62,6 +62,27 @@
   - type: Pinpointer
     component: NukeDisk
     targetName: nuclear authentication disk
+    updateTargetName: true
+  - type: MultiplePinpointer
+    modes:
+    - Off
+    - NukeDisk
+    - NuclearBombSindicate
+    - NukeOpsShuttle
+
+- type: entity
+  name: syndicate pinpointer
+  description: Produced specifically for nuclear operative missions, get that disk!
```

Comment:
the name and description are inherited from the 4-man Nuclear Operative Pinpointer. No need to name them twice.